### DISCUSSION TITLE
Provide "localized" docker images

### DISCUSSION THREAD

**Discussion by jovandeginste** [jonaswinkler/paperless-ng#886] [Feature Requests]
Currently, every time a new version of the Paperless-NG docker image is released, I build my own with the current Dockerfile:

```
FROM jonaswinkler/paperless-ng
RUN apt-get update && apt-get -y install tesseract-ocr-nld
```

Suggestion: provide pre-built docker images for every language mostly translated in Paperless-NG, based on the snippet.

No idea if there are practical issues with this idea...? If at some point, Paperless-NG is translated in too many languages for this to be practical, we may think of a better solution :wink:

**Comment by jonaswinkler** [comment]
There's also PAPERLESS_OCR_LANGUAGES (see https://paperless-ng.readthedocs.io/en/latest/configuration.html#docker-specific-options).

That will cause the docker image to install your language during startup! That happens during every start-up though, so I can clearly see the advantages of "localized" images. No idea how to do that though.

**Reply by jovandeginste** [reply]
Yes, I used that before, but that added yet another minute or so to the start-up time ...
